### PR TITLE
Move the script for installing required packages into the main repo.

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

As outlined in #7561, move the `install-required-packages.sh` script into the main Netdata repo. This puts the file where it's most likely to be seen by the people who actually need to review any changes to it.

##### Component Name

area/ci
area/docs
area/packaging

##### Additional Information

Fixes: #7561

### CODE REVIEW

**Comment by ktsaou** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/netdata/netdata/pullRequest?prid=4693753&bid=15690253)

**Comment by Ferroin** [comment]
For anyone reviewing this, the lifecycle check failures in Travis are expected, as they use the installer script, which will try to pull the file from the new location in the master branch, where it doesn't yet exist.

Long term, those need updated to pull from the local repo instead of pulling from the master branch, but that's beyond the scope of this PR.

File: .travis/package_management/configure_rpm_lxc_environment.py [hunk 1]
```diff
@@ -49,7 +49,7 @@
 print("2.1 Preparing repo on LXC container")
 common.prepare_repo(container)
 
-common.run_command(container, ["wget", "-T", "15", "-O", "/home/%s/.install-required-packages.sh" % (os.environ['BUILDER_NAME']), "https://raw.githubusercontent.com/netdata/netdata-demo-site/master/install-required-packages.sh"])
+common.run_command(container, ["wget", "-T", "15", "-O", "/home/%s/.install-required-packages.sh" % (os.environ['BUILDER_NAME']), "https://raw.githubusercontent.com/netdata/netdata/master/packageing/installer/install-required-packages.sh"])
```
Comment (knatsakis) [inline .travis/package_management/configure_rpm_lxc_environment.py]: I think there is a typo here: `packageing` instead of `packaging`

Comment (Ferroin) [reply .travis/package_management/configure_rpm_lxc_environment.py]: Indeed it is a typo. I'll have a fix pushed shortly.

**Comment by ktsaou** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/netdata/netdata/pullRequest?prid=4693753&bid=15690253)

**Comment by ktsaou** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/netdata/netdata/pullRequest?prid=4693753&bid=15690253)

**Comment by ktsaou** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/netdata/netdata/pullRequest?prid=4693753&bid=15690253)

File: packaging/installer/install-required-packages.sh [hunk 2]
```diff
@@ -0,0 +1,1772 @@
...
+
+else
+	echo >&2
+	echo >&2 "All required packages are already installed. Now proceed to the next step."
+	echo >&2
+fi
+
+remote_log "OK"
+
+exit 0
```
Comment (prologic) [inline packaging/installer/install-required-packages.sh]: This script was directly copied from somewhere else right? Can you show a `diff` of the two here?

Comment (Ferroin) [reply packaging/installer/install-required-packages.sh]: Source is https://raw.githubusercontent.com/netdata/netdata-demo-site/master/install-required-packages.sh

The scripts themselves are currently identical.

**Comment by Ferroin** [comment]
> Recent changes LGTM. I just want to ensure the script we moved is identical to the original. And can you look into those CI failures? I like to have (_green_) PR(s) :)

Unfortunately, there's not anything I can do about the failures in Travis. It's all lifecycle checks which are failing, and their failures are actually expected given that this is changing the `kickstart.sh` script to pull the script being moved by this PR from the new location, where it obviously isn't yet present. I'd actually be concerned if they were passing for this PR, because that would indicate we're doing something stupid there that doesn't match up with how user systems would be going through the process.

In fact, the sole passing one (the uncontainerized Ubuntu one at the top of the list) seems to be broken based on this...

**Comment by prologic** [comment]
> Unfortunately, there's not anything I can do about the failures in Travis. It's all lifecycle checks which are failing, and their failures are actually expected given that this is changing the `kickstart.sh` script to pull the script being moved by this PR from the new location, where it obviously isn't yet present. I'd actually be concerned if they were passing for this PR, because that would indicate we're doing something stupid there that doesn't match up with how user systems would be going through the process.
> 
> In fact, the sole passing one (the uncontainerized Ubuntu one at the top of the list) seems to be broken based on this...

Ahh my bad :) I wasn't quite understanding the impact of the change here. You are right!

**Pull request merged**